modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -550,3 +550,84 @@ RETURN largeLists
 1+d|Rows: 2
 
 |===
+
+[[composite-databases]]
+== Composite databases
+`CALL { ... } IN TRANSACTIONS` can be used on link:{neo4j-docs-base-uri}/operations-manual/{page-version}/composite-databases[composite databases].
+Composite databases are virtual databases used for querying their constituents.
+The examples above are generally valid for composite databases, but they must be slightly modified to accommodate the nature of composite databases.
+
+The following two examples show how you can use `CALL { ... } IN TRANSACTIONS` on a composite database.
+
+.friends.csv
+[source, csv, filename="friends.csv"]
+----
+1,Bill,26
+2,Max,27
+3,Anna,22
+4,Gladys,29
+5,Summer,24
+----
+
+.Create `Person` nodes on all constituents, drawing data from friends.csv 
+[source, cypher, test-skip]
+----
+UNWIND graph.names() AS graphName
+LOAD CSV FROM 'file:///friends.csv' AS line
+CALL {
+  USE graph.byName( graphName )
+  WITH line
+  CREATE (:Person {name: line[1], age: toInteger(line[2])})
+} IN TRANSACTIONS
+----
+
+.Remove all nodes from all constituents
+[source, cypher, test-skip]
+----
+UNWIND graph.names() AS graphName
+CALL {
+  USE graph.byName( graphName )
+  MATCH (n)
+  RETURN id(n) AS id
+}
+CALL {
+  USE graph.byName( graphName )
+  WITH id
+  MATCH (n)
+  WHERE id(n) = id
+  DETACH DELETE n
+} IN TRANSACTIONS
+----
+
+The two examples demonstrate another useful feature of `CALL { ... } IN TRANSACTIONS`.
+Even though composite databases allow accessing multiple graphs in a single query, only one graph can be modified in a single transaction.
+`CALL { ... } IN TRANSACTIONS` offers a way of constructing queries which modify multiple graphs.
+
+[NOTE]
```

Comment:
@stefano-ottolenghi @rsill-neo4j What do you think abut this note? Is it a reasonable way how to tell the users about known issues?